pkg/metrics/metrics.go

```diff
@@ -0,0 +1,100 @@
+package metrics
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Error to be  returned when marketplace is unable to reach quay.io.
```

Comment:
We should avoid specifically referencing quay.io whenever possible. Keep in mind that the fact that quay hosts app registry doesn't mean that it's the only way to get an app registry instance.